docs/features/theme.stories.mdx

```diff
@@ -178,36 +178,31 @@ theme.borderWidth.[kilo|mega]
 ## Typography
 
 ```
-theme.typography.[headings|subHeadings|text].<SIZE_NAME>.[fontSize|lineHeight]
+theme.typography.[headlines|subHeadlines|text].<SIZE_NAME>.[fontSize|lineHeight]
```

Comment:
These don't match the [new names](https://github.com/sumup-oss/circuit-ui/blob/typography-refresh-v2/packages/design-tokens/themes/shared.ts#L118-L149) in the design tokens yet: 

```suggestion
theme.typography.[headline|subHeadline|body].<SIZE_NAME>.[fontSize|lineHeight]
```